addons/web/static/src/views/kanban/kanban_record.js

```diff
@@ -166,30 +168,37 @@ export class KanbanRecord extends Component {
     static defaultProps = {
         colors: COLORS,
         deleteRecord: () => {},
+        getSelection: () => [],
         archiveRecord: () => {},
         openRecord: () => {},
+        selectionAvailable: false,
+        toggleSelection: () => {},
     };
     static props = [
         "archInfo",
         "canResequence?",
         "colors?",
         "Compiler?",
         "forceGlobalClick?",
+        "getSelection?",
```

Comment:
is it relevant for a kanban record to know the entire selection? I would simply add a boolean props, indicating whether this record is selected or not